perf/security/file-mount-tests/non-sds-1/setup_test.sh

```diff
@@ -12,18 +12,32 @@ if [[ ! -d "${WD}" ]]; then
 fi
 
 URL=""
-if [[ "$RELEASETYPE" == "daily" ]]; then
-  URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-elif [[ "$RELEASETYPE" == "release" ]]; then
-	URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-elif [[ "$RELEASETYPE" == "pre-release" ]]; then
-	URL="https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${RELEASE}/istio-${RELEASE}-linux.tar.gz"
-else
-  echo "Please specify RELEASETYPE"
-fi
-
-wget -O "$WD/istio-${RELEASE}-linux.tar.gz" "${URL}"
-tar xfz ${WD}/istio-${RELEASE}-linux.tar.gz -C $WD
+case "${OSTYPE}" in
+  darwin*)
+    if [[ "$RELEASETYPE" == "daily" ]]; then
+        URL="https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
+    elif [[ "$RELEASETYPE" == "release" ]]; then
+        URL="https://github.com/istio/istio/releases/download/${RELEASE}/istio-${RELEASE}-osx.tar.gz"
```

Comment:
1. Instead you can just use suffix (release)/istio-(release)- (ostype).tar.gz
2. The prefix can be templatized as well.

It would be cleaner that way.

Reply:
Moved this part into function get_release_url() in utils/get_release.sh